lib/ex_unit/lib/ex_unit/runner.ex

```diff
@@ -54,32 +55,34 @@ defmodule ExUnit.Runner do
     |> Keyword.put(:include, include)
   end
 
-  defp loop(%{modules: :async} = config, taken) do
+  defp loop(modules \\ :async, taken \\ 0, config)
```

Comment:
I personally find an antipattern to use default arguments for private functions. Can we please pass all arguments the time we call it? Thanks.